test/integration/targets/azure_rm_virtualmachine/tasks/virtualmachine.yml

```diff
@@ -81,7 +93,7 @@
       admin_password: Password123!
       short_hostname: testvm
       os_type: Linux
-      network_interfaces: testvm001
+      network_interfaces: "{{ niclist }}"
```

Comment:
instead of updating existing test, pls add new one to cover vm with multiple nic. keep existing tests to cover create vm with one nic.